ct-app/core/node.py

```diff
@@ -245,7 +245,9 @@ async def fund_channels(self):
         ]
 
         low_balances = [
-            c for c in out_opens if int(c.balance)/1e18 <= self.params.channel.min_balance
+            c
+            for c in out_opens
+            if int(c.balance) / 1e18 <= self.params.channel.min_balance
```

Comment:
Why the 1e-18?

Reply:
To get the balance in wxHOPR instead of weiHOPR.